docs/peer-testing.md

```diff
@@ -0,0 +1,170 @@
+# [Datadog .NET AppSec] Peer-testing
+
+This document describes how to test the installation of the .NET App Sec Library
+
```

Comment:
A little blurb about what peer-testing is might be nice (for me 😅 )

Reply:
fixed in a92010b64534fc8ef02f7ad7689d58fb430b4232